packages-web/badge-button/package.json

```diff
@@ -27,13 +27,13 @@
     "packagePath": "com.mendix.widget.web",
     "scripts": {
         "start": "../../node_modules/.bin/concurrently \"../../node_modules/.bin/pluggable-widgets-tools start:server --open --subprojectPath $npm_package_config_subprojectFolder\" \"npm:dev\"",
-        "dev": "../../node_modules/.bin/pluggable-widgets-tools start:ts --subprojectPath $npm_package_config_subprojectFolder",
+        "dev": "../../node_modules/.bin/pluggable-widgets-tools start:ts -- --subprojectPath $npm_package_config_subprojectFolder",
         "build": "../../node_modules/.bin/pluggable-widgets-tools build:ts --subprojectPath $npm_package_config_subprojectFolder",
         "format": "../../node_modules/.bin/pluggable-widgets-tools format --subprojectPath $npm_package_config_subprojectFolder",
         "lint": "../../node_modules/.bin/eslint --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
         "test": "../../node_modules/.bin/jest --config ../../scripts/test/jest.web.config.js",
-        "test:e2e": "export URL=https://badgebutton.mxapps.io || set URL=https://badgebutton.mxapps.io && wdio ../../configs/e2e/wdio.conf.js",
-        "test:e2e:dev": "export DEBUG=true || set DEBUG=true && wdio ../../configs/e2e/wdio.conf.js",
+        "test:e2e": "URL=https://badgebutton.mxapps.io ../../node_modules/.bin/wdio ../../configs/e2e/wdio.conf.js",
+        "test:e2e:dev": "DEBUG=true ../../node_modules/.bin/wdio ../../configs/e2e/wdio.conf.js",
```

Comment:
Does this work? or should we use https://www.npmjs.com/package/cross-env
`&& cross-env DEBUG=true wdio ./tests/e2e/wdio.conf.js",`

Reply:
This doesnt work on windows.

Reply:
Reverted it.